saleor/webhook/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy
+
+from ..account.models import ServiceAccount
+
+
+class Webhook(models.Model):
+    service_account = models.ForeignKey(
+        ServiceAccount, related_name="webhooks", on_delete=models.CASCADE
+    )
+    target_url = models.URLField("Target URL", max_length=255)
```

Comment:
Is the "Target URL" verbose name used on purpose? We rather don't use it in other model fields.